programs/marinade-finance/src/state/configure.rs

```diff
@@ -1,13 +1,13 @@
-use crate::{CommonError, ConfigMarinade, ConfigMarinadeParams, MAX_REWARD_FEE};
+use crate::{CommonError, ConfigureMarinade, ConfigureMarinadeParams, MAX_REWARD_FEE};
 use anchor_lang::prelude::*;
 use anchor_lang::solana_program::native_token::LAMPORTS_PER_SOL;
 
-impl<'info> ConfigMarinade<'info> {
+impl<'info> ConfigureMarinade<'info> {
```

Comment:
can you please undo the rename, so we present less changes to the ecosystem multisig?